src/service/poc_mobile.proto

```diff
@@ -37,6 +43,12 @@ message cell_heartbeat_req_v1 {
 
 message cell_heartbeat_resp_v1 { string id = 1; }
 
+//cell heartbeat report outputted by ingestor, tagged with received_timestamp
+message cell_heartbeat_ingest_report_v1 {
+    uint64 received_timestamp = 1;
```

Comment:
annotate with the expected unit for timestamp

Reply:
Fixed b07a1d1